org.gnome.Fragments.json

```diff
@@ -0,0 +1,48 @@
+{
+    "app-id" : "org.gnome.Fragments",
```

Comment:
Is it official GNOME project? If not, it would be better to use `com.github.haecker_felix.Fragments` here.

Reply:
I can change it, that's not a problem. However i thought it is okay to use the org.gnome domain for third party apps? For example FeedReader and Fractal are using this domain too. Do I have to change it?

Reply:
Both were added when we were figuring out what rules should be there. We're more "strict" about some things today.

Reply:
@haecker-felix I see that you used `de.haeckerfelix` for gradio. I think it definitely looks better than `com.github.haecker_felix`. ;-)

Reply:
Do you actually control haeckerfelix.de though?

Reply:
Currently not, but I had it in the past. I plan to renew the domain.

Unfortunately I can't do anything now, since my ThinkPad is dead since this morning...